src/dir.cr

```diff
@@ -93,6 +96,24 @@ class Dir
     String.new(dir).tap { C.free(dir as Void*) }
   end
 
+  def self.chdir path
+    if C.chdir(path) != 0
+      raise Errno.new("Error while changing directory")
+    end
+  end
```

Comment:
I agree. I've never checked the return value of `chdir()` in ruby outside of a repl session.